drivers/source/SFDP.cpp

```diff
@@ -26,8 +26,14 @@
 
 #if (DEVICE_SPI || DEVICE_QSPI)
 
-#include "features/frameworks/mbed-trace/mbed-trace/mbed_trace.h"
+#if MBED_CONF_MBED_TRACE_ENABLE
+#include "mbed-trace/mbed_trace.h"
 #define TRACE_GROUP "SFDP"
+#else // MBED_CONF_MBED_TRACE_ENABLE
+#define tr_info(...)
+#define tr_error(...)
+#define tr_debug(...)
+#endif // MBED_CONF_MBED_TRACE_ENABLE
```

Comment:
As something in drivers (a "core" library available for bare metal), mbed-trace shouldn't be a strict requirement. I made this change to fix bare metal build. @evedon @rajkan01